modules/core/src/test/java/org/apache/ignite/testsuites/IgniteContinuousQueryConfigVariationsSuite.java

```diff
@@ -60,12 +61,12 @@
 
     /** */
     @RunWith(IgniteContinuousQueryConfigVariationsSuite.SuiteSingleNode.class)
-    public static class SingleNodeTest {
+    @Ignore("https://issues.apache.org/jira/browse/IGNITE-11851") public static class SingleNodeTest {
```

Comment:
I suppose `@Ignore` should be placed on a separate line according to our conventions.

Reply:
done